### DISCUSSION TITLE
October 2022 Release

### DISCUSSION THREAD

**Discussion by steam0r** [cables-gl/cables_docs#370] [Show and tell]
![octoberupdate](https://user-images.githubusercontent.com/151271/195343682-47158598-1f65-457d-a4a8-9369cb42b004.png)

Hey,

we just updated cables.gl with some shiny new features. Check the [changelog](https://cables.gl/changelog) and our [blogpost](https://blog.cables.gl/2022/10/12/october-2022-release/).

Feel free to discuss all the features, give feedback, let us know what you think!

best,
team cables

**Comment by archgary** [comment]
Super Cool, still trying to figure out the WSAD thing, but the VR works. I've been waiting for this for a while and it finally come to life! Very Excited!

My tryout here [https://www.instagram.com/p/CjsHRyNDuIZ/](url). 

And was looking into ammo world to see how to combine these things together, like making the birds fall down as physical cubes with a trigger, but still struggling with it. 

Anyway, appreciate your fantastic work.
